src/Responses/DeleteStoreResponse.php

```
@@ -33,7 +33,7 @@
     public static function fromResponse(
         HttpResponseInterface $response,
         HttpRequestInterface $request,
-        SchemaValidator $validator,
+        SchemaValidatorInterface $validator,
```

PHPStan (2.1.15)

Parameter $validator of method OpenFGA\Responses\DeleteStoreResponse::fromResponse() has invalid type OpenFGA\Responses\SchemaValidator.


(class.notFound)